testing/integration_test.go

```diff
@@ -92,13 +92,15 @@ PROFILER="$HOME/cloud-profiler-nodejs/google-cloud-profiler-$VERSION.tgz"
 
 TESTDIR="$HOME/test"
 mkdir -p "$TESTDIR"
-cp "testing/busybench.js" "$TESTDIR"
-cd "$TESTDIR"
+cp -r "testing/ts-bench" "$TESTDIR"
```

Comment:
Nit: could we call the directory just "busybench"? The "ts-" prefix is not that important.

Reply:
Done.